packages/data-portal-commons/src/components/ExpandableText.tsx

```diff
@@ -25,13 +26,17 @@ export const ExpandableText: React.FunctionComponent<ExpandableTextProps> = (
     props
 ) => {
     // truncate text by default
-    const [truncated, setTruncated] = useState<boolean>(true);
+    const [truncated, setTruncated] = useState<{ [key: string]: boolean }>({});
 
     const showMoreText = props.showMoreText || 'show more';
     const showLessText = props.showLessText || 'show less';
+    const cellKey = `${props.value}`;
```

Comment:
Not sure if we should introduce the cell key inside this component. What is a cell in the context of this component?

Reply:
cellKey in our component acts as a unique identifier for each row's text value within the component's state. Since each row's text can be expanded or collapsed independently, the component needs a way to track the expand/collapse state for each row separately.